networkoperator/v1/001-egressrouter.crd.yaml-patch

```diff
@@ -0,0 +1,31 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/addresses/items/properties/ip/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/addresses/items/properties/gateway/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/redirect/properties/fallbackIP/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/redirect/properties/redirectRules/items/properties/destinationIP/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/oneOf
+  value:
+    - properties:
+        mode:
+          enum: ["Redirect"]
+      required: ["redirect"]
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/networkInterface/oneOf
+  value:
+    - required: ["macvlan"]
```

Comment:
why this?

Reply:
Oh ic. This might become an enum some day.

Reply:
same problem as for the `redirect` mode. In future we might support more types than `macvlan`, but at a time only one config can be given